tests/spark/test_spark_model_export.py

```diff
@@ -217,6 +220,22 @@ def test_mleap_output_json_format(spark_model_iris, model_path):
     assert type(json_schema["fields"][0]) == dict
     assert "name" in json_schema["fields"][0]
 
+    json_schema_str = json.dumps(json_schema)
+
+    ReflectionUtil = _jvm().py4j.reflection.ReflectionUtil
+    # Parse the json schema as a Spray JSON `JsValue` object that MLeap's
+    # schema parser can process.
+    input_creator_clazz = ReflectionUtil.classForName("spray.json.ParserInput$")
+    input_creator_inst = input_creator_clazz.getField("MODULE$").get(input_creator_clazz)
+    input_item = input_creator_inst.apply(json_schema_str)
+    input_item_js_value = _jvm().spray.json.JsonParser(input_item).parseJsValue()
+
+    # Parse the `JsValue` representation of the JSON schema. If the schema is a valid
+    # MLeap schema, we expect this method to succeed.
+    mleap_parser_clazz = ReflectionUtil.classForName("ml.combust.mleap.json.JsonSupport$")
+    mleap_parser_inst = mleap_parser_clazz.getField("MODULE$").get(mleap_parser_clazz)
+    mleap_parsed_schema = mleap_parser_inst.MleapStructTypeFormat().read(input_item_js_value)
```

Comment:
What is the point of these tests? We don't verify anything about `mleap_parsed_schema`, for example, and it seems like a lot of code that's duplicating the mleap stuff.

Reply:
Btw, I did expect to see some test which confirms that we can now serialize schemas that we couldn't before -- but just from a "this works now, and fails without this patch" perspective. We don't need to do a whole bunch of introspection on it.